lib/services/android-project-service.ts

```diff
@@ -57,12 +57,19 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 				emulatorServices: this.$androidEmulatorServices,
 				projectRoot: projectRoot,
 				deviceBuildOutputPath: path.join(projectRoot, "build", "outputs", "apk"),
-				validPackageNamesForDevice: [
-					`${packageName}-debug.apk`,
-					`${packageName}-release.apk`,
-					`${projectData.projectName}-debug.apk`,
-					`${projectData.projectName}-release.apk`
-				],
+				getValidPackageNames: (buildOptions: { isReleaseBuild?: boolean, isForDevice?: boolean }): string[] => {
+					if (buildOptions.isReleaseBuild) {
```

Comment:
Can be rewritten as:
`
const buildMode = buildOptions.isReleaseBuild ? "release" : "debug"; // or constants
return [
  `${packageName}-${buildMode}.apk`,
  `${projectData.projectName}-${buildMode}.apk` 
];
`